src/components/layers/LayerControl.svelte

```diff
@@ -1,32 +1,42 @@
 <script lang="ts">
     import L from 'leaflet';
     import { getContext, onMount, onDestroy } from 'svelte';
+
+    import type { Config, Layer } from '../config';
     import GeoJsonLayer from './GeoJsonLayer.svelte';
     import { CreateLayer } from './LayerCreator.svelte';
-    import type { Config, Layer } from '../config';
-
-    import { layersStore } from '../stores.js';
+    import { layersStore } from '../../stores';
 
     export let config: Config|undefined = undefined;
-    let control = undefined;
-    let layers = [];
-    let layersMap = {};
+
+    // FIXME types, but need to figure out leaflet types here
```

Comment:
What's wrong with the types that are there?

Reply:
I think I did actually fix this and thought I put it in another PR.